src/main/webapp/assets/style.css

```diff
@@ -0,0 +1,94 @@
+:root {
+  --dark-green: #667d51;
+  --light-teal: #abcbbc;
+  --light-green: #b9c9a7;
+  --light-maroon: #a98585;
+  --dark-maroon: #955757;
+  --peach: #f2ddc9;
+  --auburn: #ce853f;
+  --tan: #e4c492;
+  --off-white: #f2f0f0;
+}
+
+html,
+body {
+  background-color: var(--off-white);
+  font-family: 
+    'Roboto', 
+    sans-serif;
+  color: #000000;
+  font-size: 1.2rem;
+  margin: 0;
+  padding: 0;
+  min-width: 300px;
+}
+
+header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  height: 5rem;
+}
+
+.logo {
+  letter-spacing: 1px;
+  font-size: larger;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-around;
+  padding: 0px;
+}
+
+.navlink {
+  list-style: none;
+  margin: 1rem;
+}
+
+.navlink a {
+  text-decoration: none;
+  color: #000000;
+}
+
+.navlink a:hover {
+  color: var(--dark-green);
+}
```

Comment:
What do you think about grouping related rules with a comment? Like 
```
//
// Navbar
//

.nav {
...
}

.navlink {
...
}
```
That way, when our css file eventually gets pretty large, it'll be easier to find relevant rules! If not, no worries though, we can keep it as it :0

Reply:
Yes! I'll add that right now!